x/ethbridge/keeper/msg_server.go

```diff
@@ -246,8 +246,8 @@ func (srv msgServer) UpdateWhiteListValidator(goCtx context.Context,
 	return &types.MsgUpdateWhiteListValidatorResponse{}, nil
 }
 
-func (srv msgServer) UpdateCethReceiverAccount(goCtx context.Context,
-	msg *types.MsgUpdateCethReceiverAccount) (*types.MsgUpdateCethReceiverAccountResponse, error) {
+func (srv msgServer) UpdateNativeTokenReceiverAccount(goCtx context.Context,
```

Comment:
Change to `UpdateCrossChainFeeReceiverAccount`

Reply:
renamed